<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WidgetMenuPanel</class>
 <widget class="QWidget" name="WidgetMenuPanel">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1011</width>
    <height>763</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color:black;
color:white;
font-family:&quot;맑은 고딕&quot;;
}
QWidget:title
{
    background-color:black; color:white;
}</string>
  </property>
  <widget class="QToolButton" name="toolButton_6">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:#494949;
	color:white;
}

QToolButton:hover {
	background-color:#FF6B27;
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Zoom Out</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_3">
   <property name="geometry">
    <rect>
     <x>303</x>
     <y>50</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:black;
	color:white;
}

QToolButton:hover {
	color:#FF6B27;
}</string>
   </property>
   <property name="text">
    <string>View</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>726</x>
     <y>50</y>
     <width>251</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:black;
	color:white;
}

QToolButton:hover {
	color:#FF6B27;
}</string>
   </property>
   <item>
    <property name="text">
     <string>Supplement</string>
    </property>
   </item>
  </widget>
  <widget class="QToolButton" name="toolButton_2">
   <property name="geometry">
    <rect>
     <x>511</x>
     <y>50</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:black;
	color:white;
}

QToolButton:hover {
	color:#FF6B27;
}</string>
   </property>
   <property name="text">
    <string>Tool</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_4">
   <property name="geometry">
    <rect>
     <x>104</x>
     <y>50</y>
     <width>81</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:black;
	color:white;
}

QToolButton:hover {
	color:#FF6B27;
}</string>
   </property>
   <property name="text">
    <string>File</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_8">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>210</y>
     <width>300</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:#494949;
	color:white;
}

QToolButton:hover {
	background-color:#FF6B27;
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Save File</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:#494949;
	color:white;
	position: relative;
}

QToolButton:hover {
	background-color:#FF6B27;
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Zoom In</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>2</x>
     <y>140</y>
     <width>1001</width>
     <height>16</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border:1px solid #FF6B27;</string>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_9">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>300</y>
     <width>971</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:#494949;
	color:white;
	position: relative;
}

QToolButton:hover {
	background-color:#FF6B27;
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Subsidiary Menu</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_10">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>390</y>
     <width>971</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:#494949;
	color:white;
	position: relative;
}

QToolButton:hover {
	background-color:#FF6B27;
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Direct Change</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_11">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>480</y>
     <width>971</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:#494949;
	color:white;
	position: relative;
}

QToolButton:hover {
	background-color:#FF6B27;
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Current Position</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_12">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>570</y>
     <width>971</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:#494949;
	color:white;
	position: relative;
}

QToolButton:hover {
	background-color:#FF6B27;
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Save Data</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_13">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:#494949;
	color:white;
	position: relative;
}

QToolButton:hover {
	background-color:#FF6B27;
	color:white;
}</string>
   </property>
   <property name="text">
    <string>Function Key</string>
   </property>
  </widget>
  <widget class="QToolButton" name="toolButton_14">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>660</y>
     <width>481</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QToolButton {
	background-color:#494949;
	color:white;
	position: relative;
}

QToolButton:hover {
	background-color:#FF6B27;
	color:white;
}</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="doubleSpinBox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>210</y>
     <width>201</width>
     <height>80</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">QDoubleSpinBox{
color:white;
border:1px solid #494949;
background-color:black;
}

QDoubleSpinBox::up-button {
	width:30px;
}

QDoubleSpinBox::down-button {
	width:30px
}</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>170</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Zoom Increment</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
